packages/common/src/items/createResource.ts

```diff
@@ -0,0 +1,71 @@
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
+import { addItemResource } from "@esri/arcgis-rest-portal";
+import HubError from "../HubError";
+import { objectToJsonBlob, stringToBlob } from "../resources";
+import { getPortalApiUrl } from "../urls";
+
+/**
+ * Given an item, and owner, add a resource to the item and returns its url
+ *
+ * @export
+ * @param {string} id
+ * @param {string} owner
+ * @param {*} resource
+ * @param {string} resourceName
+ * @param {IUserRequestOptions} ro
+ * @param {string} [prefix=""]
+ * @return {*}  {Promise<string>}
+ */
+export async function createResource(
```

Comment:
I'd name this `addResource` for consistency with the `arcgis-rest-portal` function that this wraps